CONTRIBUTING.md

```diff
@@ -243,3 +245,55 @@ $ make unit TEST_UNIT_PACKAGE=github.com/elastic/cloud-sdk-go/pkg/api
 -> Running unit tests for cloud-sdk-go...
 ok      github.com/elastic/cloud-sdk-go/pkg/api 1.277s  coverage: 89.0% of statements
 ```
+
+### Generating mocks
```

Comment:
I don't think we've talked about introducing these for the entire SDK 🤔 let's leave it for the logging package only for now :)
```suggestion
### Generating mocks for the logging package
```

Reply:
ok it sounds weird who want to contribute to use `mockery` only for a single package and especially a package that has already the mocks generated and it's changed once in a year. 
Any reason we don't want to use `mockery`?

Also the `generate-mocks` make target is not restricted to this package. It can generate mocks for any given package, so it's a bit confusing to have this as part of our documentaiton

Reply:
I see what you mean about how it might sound strange to recommend using for a single package, I was talking specifically about this sentence:
> Any new mocks should be generated with Mockery

While I don't want to block tooling from adding mocks in whichever way you find most suitable to test the logging package, which is mostly maintained/used by tooling. I also do not think that we should commit to only using Mockery for all of our mocks. My reasons are as follows:
- For our apivalidation tests we are already using [prism](https://github.com/stoplightio/prism).
- As far as I know no one in delivery has dug too deeply in Mockery to confirm at this moment that we are OK committing to using Mockery exclusively.

If you change the above sentence to say something along the lines of `An option to generate mocks is to use Mockery via the make generate-mocks target...` I would be OK with that 🙂  .